.docker/Dockerfile-template.ckan

```diff
@@ -26,9 +26,8 @@ COPY vars/shared-{VARS_TYPE}.var.yml "${APP_DIR}"/bin/extensions.yml
 
 RUN chmod +x "${APP_DIR}"/bin/*.sh /usr/bin/ckan_cli
 
-ENV DOCKERIZE_VERSION=v0.6.1
-RUN wget -O - https://github.com/jwilder/dockerize/releases/download/${DOCKERIZE_VERSION}/dockerize-linux-amd64-${DOCKERIZE_VERSION}.tar.gz \
-        | tar -C /usr/local/bin -xzvf -
+RUN which ps || apt-get install -y procps
```

Comment:
not alpine compatible but as you are fixing to ubuntu only images, this is ok.

Reply:
It actually should be compatible, because the old Alpine containers provide `ps` so the second half won't execute.